config/routes.rb

```diff
@@ -55,5 +55,7 @@
   #   end
 
   root 'home#index'
-  resources :cards
+  resources :cards do
+    patch 'check_words', on: :member
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.